.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: ruby
 
 before_install:
-  - gem update --system
```

Comment:
@olleolleolle are you sure this is taken care of automatically? My experience in other repos that use Travis, this was something that had to be done as it does update system gems like RubyGems for example.